myosuite/envs/myo/myochallenge/bimanual_v0.py

```diff
@@ -474,5 +475,5 @@ def evaluate_contact_trajectory(contact_trajectory: List[set]):
         return ContactTrajIssue.PROSTH_SHORT
 
     # Check if only goal was touching object for the last CONTACT_TRAJ_MIN_LENGTH frames
-    elif not np.all([{ObjLabels.GOAL} == s for s in contact_trajectory[-CONTACT_TRAJ_MIN_LENGTH:]]):
+    elif not np.all([{ObjLabels.GOAL} == s for s in contact_trajectory[-GOAL_CONTACT + 2:]]):
```

Comment:
can you add a comment why to start from `-GOAL_CONTACT + 2`?

Reply:
added comment here  01a4adc, basically a buffer zone around trajectory evaluation